lib/db.js

```diff
@@ -419,19 +461,38 @@ module.exports = (
 
   DB.prototype.updateSessionToken = function (token, userAgentString) {
     log.trace({ op: 'DB.updateSessionToken', uid: token && token.uid })
-
-    if (! token.update(userAgentString)) {
+    if (! token.update(userAgentString) || ! this.redis) {
       return P.resolve()
     }
 
-    return this.pool.post('/sessionToken/' + token.id + '/update', {
+    const uid = token.uid
+    const newToken = [{
+      tokenId: token.id,
+      uid: uid,
       uaBrowser: token.uaBrowser,
       uaBrowserVersion: token.uaBrowserVersion,
       uaOS: token.uaOS,
       uaOSVersion: token.uaOSVersion,
       uaDeviceType: token.uaDeviceType,
-      lastAccessTime: token.lastAccessTime
-    })
+      lastAccessTime: token.lastAccessTime,
+      createdAt: token.createdAt
+    }]
+    let sessionTokens = []
+    if (features.isLastAccessTimeEnabledForUser(uid)) {
```

Comment:
UPDATE: so remove `lastAccessTime: marshallLastAccessTime(item.lastAccessTime, uid, item.email),` from devices. Otherwise when we go from 100% to 30% the 70% will have a blank "start syncing" string